DevOps/CI-github-actions.adoc

```diff
@@ -0,0 +1,161 @@
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:toc:
+:toc-placement!:
+
+= Github Actions
+
+toc::[]
+
+This document in no way indicates that Github Actions is better than another solution: it just allows you to present the Github Actions service
```

Comment:
Pq “it just allows you” ? Qqch comme “it only describes…” me semblerait mieux ici.

Reply:
Fix a38d61e96d261fec0734b4560b97b0ccacda5ebd